In [1]:
import os
from pathlib import Path

PROJECT_ROOT = Path(r"E:\ifrs9-home-credit-pd-model")
RAW_DIR = PROJECT_ROOT / "data" / "raw"

RAW_DIR.mkdir(parents=True, exist_ok=True)

print("Project root:", PROJECT_ROOT)
print("Raw data folder:", RAW_DIR)
print("Kaggle token exists:", os.getenv("KAGGLE_API_TOKEN") is not None)

Project root: E:\ifrs9-home-credit-pd-model
Raw data folder: E:\ifrs9-home-credit-pd-model\data\raw
Kaggle token exists: True


In [2]:
import kagglehub

path = kagglehub.competition_download(
    "home-credit-default-risk"
)

print(path)

KaggleApiHTTPError: 401 Client Error.

You don't have permission to access resource at URL: https://kaggle.com/competitions/home-credit-default-risk
Please make sure you are authenticated and have accepted the competition rules which can be found at this location: https://kaggle.com/competitions/home-credit-default-risk/rules

This notebook requires Kaggle API credentials to download the dataset.
If credentials are missing, the download step returns a 401 authorization error.